From d92c5521932ae952e2e729cb80302bdc334c814c Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Fri, 16 Nov 2012 09:41:06 +0000
Subject: [PATCH] libXt: util: don't link makestrs with target cflags

The line:

AM_CFLAGS = $(XT_CFLAGS)

in util/Makefile.am is wrong because it adds target cflags to the
compilation of makestrs, which is built for the build machine, which
leads to build failures when cross-compiling.

We also remove the inclusion of <X11/Xos.h> from makestrs.c, because
it was the only non-standard header being included (and therefore
possibly requiring special cflags), but it was in reality not useful
at all to build makestrs.c.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 util/Makefile.am | 1 -
 util/makestrs.c  | 1 -
 2 files changed, 2 deletions(-)

diff --git a/util/Makefile.am b/util/Makefile.am
index 800b35b..f2dd1f9 100644
--- a/util/Makefile.am
+++ b/util/Makefile.am
@@ -11,7 +11,6 @@ EXTRA_DIST = \
 	StrDefs.ht \
 	string.list
 
-AM_CFLAGS = $(XT_CFLAGS)
 makestrs_SOURCES = makestrs.c
 
 
diff --git a/util/makestrs.c b/util/makestrs.c
index a52866a..00c861f 100644
--- a/util/makestrs.c
+++ b/util/makestrs.c
@@ -27,7 +27,7 @@ in this Software without prior written authorization from The Open Group.
 /* Constructs string definitions */
 
 #include <stdio.h>
-#include <X11/Xos.h>
+#include <string.h>
 #include <stdlib.h>
 #include <unistd.h>
 
-- 
1.8.1

